assets/sass/local.sass

```diff
@@ -1,2 +1,6 @@
 .govuk-main-wrapper
   min-height: 600px
+
+.cas3-phase-banner
+  border-bottom: 0
+  border-top: 1px solid $govuk-border-colour
```

Comment:
Worth creating a component override like https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/main/assets/sass/components/_filter.scss? Maybe not. I'm easy 😃